src/components/slack/list/slacklist.component.ts

```diff
@@ -500,6 +511,19 @@ export class SlackListComponent implements OnInit, OnDestroy {
         }
     }
 
+    onMouseEnterReaction(reaction: DisplaySlackReactionInfo) {
+        reaction.showReactionUsers = true;
```

Comment:
Okay I added the clearTimeout. It will work fine if any changes are detected in waiting for showing reaction users.